### PR TITLE
Update some receiver/exporter docs

### PR DESCRIPTION
**Pull request by flands**
* Jaeger
* OpenCensus
* OTLP

### CODE REVIEW

File: receiver/otlpreceiver/README.md [hunk 1]
```diff
@@ -1,14 +1,42 @@
...
+- `tls_credentials` (default = unset): configured the receiver to use TLS. See
+  TLS section below.
+
+Examples:
+
 ```yaml
 receivers:
   otlp:
+  otlp/withendpoint:
+    endpoint: 127.0.0.1:55680
```
Comment (bogdandrutu) [inline receiver/otlpreceiver/README.md:39]: Use `0.0.0.0:55680`?

Comment (flands) [reply receiver/otlpreceiver/README.md:39]: `0.0.0.0:55680` is the default configuration -- this shows how to change it to something non-default.

File: receiver/opencensusreceiver/README.md [hunk 2]
```diff
@@ -1,15 +1,42 @@
...
+  https://godoc.org/google.golang.org/grpc/keepalive#ServerParameters for more
+  information
+  - `MaxConnectionIdle` (default = infinity)
+  - `MaxConnectionAge` (default = infinity)
+  - `MaxConnectionAgeGrace` (default = infinity)
+  - `Time` (default = 2h)
+  - `Timeout` (default = 20s)
+- `max_recv_msg_size_mib` (default = infinity): sets the maximum size of messages accepted
+- `max_concurrent_streams`: sets the limit on the number of concurrent streams
+- `tls_credentials` (default = unset): configured the receiver to use TLS. See
```
Comment (pmcollins) [inline receiver/opencensusreceiver/README.md:30]: Did you mean to type `configures` here?

File: receiver/otlpreceiver/README.md [hunk 3]
```diff
@@ -1,14 +1,42 @@
...
+  https://godoc.org/google.golang.org/grpc/keepalive#ServerParameters for more
+  information
+  - `MaxConnectionIdle` (default = infinity)
+  - `MaxConnectionAge` (default = infinity)
+  - `MaxConnectionAgeGrace` (default = infinity)
+  - `Time` (default = 2h)
+  - `Timeout` (default = 20s)
+- `max_recv_msg_size_mib` (default = infinity): sets the maximum size of messages accepted
+- `max_concurrent_streams`: sets the limit on the number of concurrent streams
+- `tls_credentials` (default = unset): configured the receiver to use TLS. See
```
Comment (pmcollins) [inline receiver/otlpreceiver/README.md:30]: configures?

File: receiver/opencensusreceiver/README.md (hunk 2 above)
Comment (flands) [inline receiver/opencensusreceiver/README.md:30]:
```suggestion
- `tls_credentials` (default = unset): configures the receiver to use TLS. See
```

File: receiver/otlpreceiver/README.md (hunk 3 above)
Comment (flands) [inline receiver/otlpreceiver/README.md:30]:
```suggestion
- `tls_credentials` (default = unset): configures the receiver to use TLS. See
```

**Comment by bogdandrutu** [comment]
@pmcollins PTAL

**Comment by bogdandrutu** [comment]
Please rebase

**Pull request merged**